### ISSUE TITLE
New Role: Armor Supplier

### ISSUE THREAD

**Issue by Histalek** [TTT-2/TTT2#371]
* once the Armor System is in place a role that supplies armor to players could be nice

* current ideas:
  * 'Rook'-like (Rainbow6Siege Operator), place a stationary 'armor-station' which upon interacting gives the interacting player armor once
  * 'Torbjörn'-like (Overwatch Character), throws an entity which upon contact with a player gives the player armor

**Comment by Lunethia** [comment]
- I'm more in favour of a Torbjörn-like varient if it were to be an innocent role, since it makes you able to choose which player you give your armor to. So for example as an innocent role he might buff up known innocents and provide them with some better survivability for the team

(A detective/sniffer or revealed survivalist/priest come to mind).

- Though another take on it would be the rook-like variant as an evil role (team traitor) who would be an additional traitor role with a supportive ability to make life easier for his fellow traitors.
The idea behind choosing this design over the other when used as an "evil" role, would be the risk included with placing down the "armor station/pack/whatever we end up with", as you could be spottet by a non-traitor palyer by doing so, and also that non-traitor players who eventually find said "armor station" couls use it as well, if intendet so.

(A different mechanic could be that it is traitor only, but that would mean that it had to be included in additional traitor roles in the future etc, though i'd probably prefer this over the other ^^)

- Additionally I would suggest choosing a different name other than "Armor Supplier", once a more distinct design has been made out, to fit the team the role (or roles!!) end up in.

**Comment by Histalek** [comment]
Yea the name is definetly a placeholder, maybe i should have stated that =D

i wanted to start a discussion because i don't really had a goal for the role in mind, besides just giving out armor i mean. 
I like both your ideas, but i feel like it still misses some gameplay besides being an armor dispenser ^^

**Comment by TimGoll** [comment]
I like the innocent player that drops single armor perks for others to pick up, but I like the armor station too. I have no idea for a cool name though

**Comment by lebroomer** [comment]
I think the concept could fit better as a Class instead of a Role, since it has no real connection to the Innocent team.

**Comment by TimGoll** [comment]
Yes. But since not everybody wants to play with classes, an additional role that replaces an innocent player is a good idea

or - what about an detective item? Every detective (and sniffer) will get it automatically on spawn
